# HG changeset patch
# User Andrew John Hughes <ahughes@redhat.com>
# Date 1317357774 -3600
# Node ID 15866ac3f41bc8528c19a4c14190cbca9fbef1d3
# Parent  875cada723d75edfc9b26973ac71a7317d39ee49
Turn off forced -Werror when building com.sun.java.pack

2011-09-30  Andrew John Hughes  <ahughes@redhat.com>

	Bump changesets to bring in fix which
	turns off forced -Werror.
	* Makefile.am:
	(CORBA_CHANGESET): Updated.
	(HOTSPOT_CHANGESET): Likewise.
	(JAXP_CHANGESET): Likewise.
	(JAXWS_CHANGESET): Likewise.
	(JDK_CHANGESET): Likewise.
	(LANGTOOLS_CHANGESET): Likewise.
	(OPENJDK_CHANGESET): Likewise.
	(CORBA_SHA256SUM): Likewise.
	(HOTSPOT_SHA256SUM): Likewise.
	(JAXP_SHA256SUM): Likewise.
	(JAXWS_SHA256SUM): Likewise.
	(JDK_SHA256SUM): Likewise.
	(LANGTOOLS_SHA256SUM): Likewise.
	(OPENJDK_SHA256SUM): Likewise.

diff -r 875cada723d7 -r 15866ac3f41b ChangeLog
--- a/ChangeLog	Fri Sep 30 04:33:46 2011 +0100
+++ b/ChangeLog	Fri Sep 30 05:42:54 2011 +0100
@@ -1,3 +1,23 @@
+2011-09-30  Andrew John Hughes  <ahughes@redhat.com>
+
+	Bump changesets to bring in fix which
+	turns off forced -Werror.
+	* Makefile.am:
+	(CORBA_CHANGESET): Updated.
+	(HOTSPOT_CHANGESET): Likewise.
+	(JAXP_CHANGESET): Likewise.
+	(JAXWS_CHANGESET): Likewise.
+	(JDK_CHANGESET): Likewise.
+	(LANGTOOLS_CHANGESET): Likewise.
+	(OPENJDK_CHANGESET): Likewise.
+	(CORBA_SHA256SUM): Likewise.
+	(HOTSPOT_SHA256SUM): Likewise.
+	(JAXP_SHA256SUM): Likewise.
+	(JAXWS_SHA256SUM): Likewise.
+	(JDK_SHA256SUM): Likewise.
+	(LANGTOOLS_SHA256SUM): Likewise.
+	(OPENJDK_SHA256SUM): Likewise.
+
 2011-09-30  Andrew John Hughes  <ahughes@redhat.com>
 
 	* Makefile.am:
diff -r 875cada723d7 -r 15866ac3f41b Makefile.am
--- a/Makefile.am	Fri Sep 30 04:33:46 2011 +0100
+++ b/Makefile.am	Fri Sep 30 05:42:54 2011 +0100
@@ -2,21 +2,21 @@
 
 OPENJDK_VERSION = b147
 
-CORBA_CHANGESET = d034cc90ecc2
-HOTSPOT_CHANGESET = 7693eb0fce1f
-JAXP_CHANGESET = b8d01501956a
-JAXWS_CHANGESET = e6cd09c7ef22
-JDK_CHANGESET = 7ec1845521ed
-LANGTOOLS_CHANGESET = fb7fb3071b64
-OPENJDK_CHANGESET = 3defd24c2671
+CORBA_CHANGESET = 953de8c7bccb
+HOTSPOT_CHANGESET = b28ae681bae0
+JAXP_CHANGESET = 948e734135ea
+JAXWS_CHANGESET = 5d3734549424
+JDK_CHANGESET = d9fca71ba183
+LANGTOOLS_CHANGESET = 9b85f1265346
+OPENJDK_CHANGESET = 0a76e5390e68
 
-CORBA_SHA256SUM = 06e7778aa1670f064e2c9fef7bdedc66fc679262ab8ee7aee018329e1ec787a8
-HOTSPOT_SHA256SUM = 6b2c9b21918183a383ab380c57f70d4be6199df0e2127bb762a20b6eca39e3a2
-JAXP_SHA256SUM = 4e9c1a2f240cee406d56832e2e7be254a87b511ee781463a95da4f000c188181
-JAXWS_SHA256SUM = 3d13408b4a81111fd88fce043ca2c96c4d31da9aa812a8d608b56248ecac4aca
-JDK_SHA256SUM = 843e7ccd4c2bad5c96aaa983e34f1eb27510229f138115fa0951e05358c50b2b
-LANGTOOLS_SHA256SUM = 9ddc00ec50fd2f5e331dc2bc10da4e23b69bf644eb92d50b39a2003c18fb5aa1
-OPENJDK_SHA256SUM = 4043a75c2c4385dd735f8dbbf2369311ce1b951217c9dbe9bba9609e24eb291e
+CORBA_SHA256SUM = 303d862f722d34ede330e9afdb0f2c9d61e02134beb4d562957e9574a91f8cca
+HOTSPOT_SHA256SUM = dcb5a8d4a0a466e3673f891cea40ce163c02f26b0054dfc41c30e0f87c5c2f64
+JAXP_SHA256SUM = 66948dee25e0224b12587ba150b21dab2e21b260a853bd1272e07c8d08d2e586
+JAXWS_SHA256SUM = 6aace2cc9f1a98a3822733ad568cc070ec178ff3618c05229c68072af3ed4765
+JDK_SHA256SUM = d8e9bb264f2a31424764466273f10c05d4247328ab4c08a0f4b8123993d78e65
+LANGTOOLS_SHA256SUM = 03a256afc7371b3b0fbbbfd80a318e22984f6cbff26082948e8d5845ba193aee
+OPENJDK_SHA256SUM = aa3b3cf17081cde2a4014e5dffec75f5b5402ad48f7f9943c32a2fa4544a8746
 
 CACAO_VERSION = 4549072ab2de
 CACAO_SHA256SUM = ec097ce96f047abbaeab4faab752c121d29f08bdeae358038a0b7cde03361d07
